src/EditorFeatures/DiagnosticsTestUtilities/ChangeSignature/ChangeSignatureTestState.cs

```diff
@@ -29,12 +31,18 @@ internal sealed class ChangeSignatureTestState : IDisposable
         public string ErrorMessage { get; private set; }
         public NotificationSeverity ErrorSeverity { get; private set; }
 
-        public static ChangeSignatureTestState Create(string markup, string languageName, ParseOptions parseOptions = null)
+        public static ChangeSignatureTestState Create(string markup, string languageName, ParseOptions parseOptions = null, OptionsCollection options = null, bool xmlMarkup = false)
         {
-            var workspace = languageName == LanguageNames.CSharp
+            var workspace = xmlMarkup ? TestWorkspace.Create(markup, composition: s_composition) :
```

Comment:
I know this is only test code, but can't "xml" be passed in for `languageName`? Its a bit disingenuous, but having a new parameter at the end that controls whether a parameter at the stat is used seems odd too.

Also this would be much more readable as a switch expression, than nested ternary.